<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sistema de Registro</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>661</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Inserção</string>
     </attribute>
     <widget class="QLabel" name="lblProduto">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Produto:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtProduto">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblPreco">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Preço:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPreco">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblTamanho">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tamanho:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTam">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lblQuant">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantidade:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtQuant">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSalvar">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtForn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblForn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fornecedor:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSalvarForn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Salvar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Busca</string>
     </attribute>
     <widget class="QLabel" name="lblBusca">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Produto:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBusca">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBusca">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>371</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Remoção</string>
     </attribute>
     <widget class="QLabel" name="lblRemove">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Produto:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRemove">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemover">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblRemForn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Fornecedor:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRemForn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemForn">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remover</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Lista de Produtos</string>
     </attribute>
     <widget class="QTextBrowser" name="txtAreaProd">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>381</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnImprime">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Atualizar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
